### PR TITLE
Add initial distance sensor code. 

### PR DESCRIPTION
**Pull request by ryanpfister**
Added the Distance Sensor (pid) . From line 61-71.

### CODE REVIEW

**Comment by CodingKoopa** [comment]
I will review this when we have a better idea of how we will exactly integrate the distance sensor into our robot's strategy.

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -57,7 +57,19 @@
...
+  private static final double kHoldDistance = 3.0;
+  // factor to convert sensor values to a distance in inches
+  private static final double kValueToInches = 0.125;
+  // proportional speed constant
+  private static final double kP = 0.05;
+  private static final int kLeftMotorPort = 0;
+  private static final int kRightMotorPort = 1;
+  private static final int kUltrasonicPort = 0;
+  private final AnalogInput m_ultrasonic = new AnalogInput(kUltrasonicPort);
+// Concludes Distance Sensor Code
```
Comment (RyanStyron) [inline src/main/java/frc/robot/Robot.java:72]: Remove this line for consistency purposes.

**Review by RyanStyron — COMMENTED** [review]
Nice contribution to the start of the distance sensor code.

**Comment by RyanStyron** [comment]
Closed due to a new pull requests initializing the same code and more.

**Pull request closed without merging**